brage-import/src/test/java/no/sikt/nva/brage/migration/lambda/BrageTestRecord.java

```diff
@@ -0,0 +1,24 @@
+package no.sikt.nva.brage.migration.lambda;
+
+import no.sikt.nva.brage.migration.testutils.NvaBrageMigrationDataGenerator;
+import no.sikt.nva.brage.migration.testutils.NvaBrageMigrationDataGenerator.Builder;
+import no.unit.nva.model.Publication;
+
+public class BrageTestRecord {
+
```

Comment:
Have no idea for better object name, but used to wrap/store existing publication in nva with brage record